llvm/include/llvm/CodeGen/GlobalMerge.h

```diff
@@ -28,6 +28,10 @@ struct GlobalMergeOptions {
   bool MergeConst = false;
   /// Whether we should merge global variables that have external linkage.
   bool MergeExternal = true;
+  /// Whether we should merge global variables that have private linkage.
+  bool MergePrivateGlobals = false;
```

Comment:
MergePrivateGlobals is unused?

Reply:
+1

Like the internal global variables, private global variables should always be merged and should not be guarded under an option. You've already done this in https://github.com/llvm/llvm-project/pull/101222 :)